docs/v3.x/plugins/graphql.md

```diff
@@ -36,6 +36,20 @@ strapi install graphql
 
 Then, start your app and open your browser at [http://localhost:1337/graphql](http://localhost:1337/graphql). You should see the interface (**GraphQL Playground**) that will help you to write GraphQL query to explore your data.
 
+## Authentication
+
+To perform authorized requests, you must first get a JWT:
+
+```graphql
+mutation {
+  login(input: { identifier: "email", password: "password" }) {
+    jwt
+  }
+}
+```
+
+Then on each request, send along an `Authorization` header in the form of `{ Authorization: "Bearer YOUR_JWT_GOES_HERE" }`. This can be set in the HTTP Headers section of your GraphQL Playground.
+
 ::: tip
```

Comment:
Can you do me a favor and remove the modheader tip from these docs, I believe it's left over from when we were using a different GQL UI than the playground.